merkle/hstar2_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // This root was calculated with the C++/Python sparse Merkle tree code in the
 // github.com/google/certificate-transparency repo.
-const sparseEmptyRootHashB64 = "xmifEIEqCYCXbZUz2Dh1KCFmFZVn7DUVVxbBQTr1PWo="
+var sparseEmptyRootHashB64 = testonly.MustDecodeBase64("xmifEIEqCYCXbZUz2Dh1KCFmFZVn7DUVVxbBQTr1PWo=")
```

Comment:
Please clarify? 

- This doesn't change the behavior of the test as is - so it passes. 
- If we changed the test only hasher, yes, this would break.  A stronger test would compute this properly based on the hasher, but there's not enough documentation on how the test vector was computed to do so.